Tests/TestBase.gd

```diff
@@ -45,7 +48,8 @@ func before_each():
 	_player_stat_component.get_stats().ready_entity_stats()
 	_enemy_stat_component.get_stats().ready_entity_stats()
 	_enemy_2_stat_component.get_stats().ready_entity_stats()
-	
+
+	_player_energy_component.ignore_cost = true
```

Comment:
why ? this is a test, you shouldn't ignore card cost

Reply:
Now that I think about it, its not doing anything right now. Players first need to queue the card and then play the card.
I'm guessing the cards that are used in the tests skip queuing the cards.

Reply:
you don't interact physically with the cards indeed, so they are not queued

Reply:
ah the idea was the allow other tests to continue working without having to worry about energy cost ?

Reply:
Yes, those tests are testing for other stuff and not the energy system directly.

Reply:
yes so it's logical to add this line indeed, just need to overwrite that in the init of the energy tests